### DISCUSSION TITLE
Why SecurityOnion switched from Grafana to Influxdb

### DISCUSSION THREAD

**Discussion by giomke** [Security-Onion-Solutions/securityonion#12982] [General] [answered]
Hello folks,
I'm very curious about this decision. Do you guys know what the purpose was behind changing from Grafana to InfluxDB?

**Comment by TOoSmOotH** [comment, marked as answer]
Main reason was license incompatibility with Google Cloud. Secondary reason was all of the dashboards had to be rewritten to work with the new version of influxdb. It made sense to get rid of a container and write those dashboards in influxdb.
